File: day005/day005.java
public class Day005 {
	
	public static void main(String[] args) {
		java.util.Scanner kb = new java.util.Scanner(System.in);
		int x, y, temp;
		System.out.print("Bir sayı giriniz: X ");
		x = kb.nextInt();
		System.out.print("Başka bir sayı daha giriniz: Y ");
		y = kb.nextInt();
		temp = x;
		x = y;
		y = temp;
		System.out.println("x: " + x + " y: " + y);
	}
}

File: day005/day005.py
x = input("İstediğiniz bir şeyi giriniz: ")
y = input("Bir şey daha giriniz: ")

temp = x
x = y
y = temp

print(f"Girdiğiniz değerlerin yeri değişti. İlk değer: {x} İkinci değer: {y}")
